analyzer/language/golang/binary/binary_test.go

```diff
@@ -43,7 +43,6 @@ func Test_gobinaryLibraryAnalyzer_Analyze(t *testing.T) {
 		{
 			name:      "broken elf",
 			inputFile: "testdata/broken_elf",
-			wantErr:   "unexpected EOF",
```

Comment:
sorry for bother. 
`wantErr` is never used, right?

Reply:
Thanks. You're right. I've removed it.